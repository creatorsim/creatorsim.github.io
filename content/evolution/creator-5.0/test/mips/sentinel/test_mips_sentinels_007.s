
#
# ARCOS.INF.UC3M.ES
# BY-NC-SA (https://creativecommons.org/licenses/by-nc-sa/4.0/deed.es)
#


.text

	main: 
			
			li $k0, 5
            jal test

            # exit
            li $v0, 10
            syscall


	test:
            # crear "stack frame" para $ra, $fp y una variable local
            
			li $k0, 5

            # return $v0
           jr $ra
