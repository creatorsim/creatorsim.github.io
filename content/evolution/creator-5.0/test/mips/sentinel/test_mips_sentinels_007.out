
cr[PC]:0x10; ir[2,v0]:0xa; ir[26,k0]:0x5; ir[31,ra]:0x8; keyboard[0x0]:''; display[0x0]:''; 

